doc/Tribler development on OS X.md

```diff
@@ -0,0 +1,123 @@
+This page contains information about setting up a Tribler development environment on OS X. Unlike Linux based systems where installing third-party libraries is often a single `apt-get` command, installing and configuring the necessary libraries requires more attention on OS X. This guide has been tested with OS X 10.10.5 (Yosemite) but should also work for OS X 10.11 (El Capitan).
+
+## Introduction
+Compilation of C/C++ libraries should be performed using Clang which is part of the Xcode Command Line Tools. The Python version shipped with OS X can be used and this guide has been tested using Python 2.7. The current installed version and binary of Python can be found by executing:
+
+```
+python --version # gets the python version
+which python # prints the path of the Python executable
+```
+
+Note that the default location of third-party Python libraries (for example, installed with `pip`) can be found in `/Library/Python/2.7/site-packages`.
+
+Many packages can be installed by using the popular brew and pip executables. Brew and pip can be installed by using:
+
+```
+ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+sudo easy_install pip
```

Comment:
You should invoke the commands with `sudo` since Python is by default located in a root-protected directory. One workaround for this is using a virtualenv setup. I will write this in the guide.